packages/wordpress-kit/src/lib/tailwindcssPlugin/utilities/Fonts.ts

```diff
@@ -0,0 +1,74 @@
+import { ThemeType } from '../../../types/TailwindcssPlugin';
+import { fontSizeList } from '../Constants';
+
+/**
+ * Class to generate all the fonts related utilities
+ * @class FontsUtilities
+ * @param {ThemeType} theme - The theme object from tailwindcss plugin
+ *
+ * @example
+ * ```
+ * const fonts = new FontsUtilities(theme);
+ *
+ * fonts.getFontSizeUtilities() // returns the font size utilities
+ * ```
+ */
+export class FontsUtilities {
+  private theme: ThemeType;
+
+  constructor(theme: ThemeType) {
+    this.theme = theme;
+  }
+  /**
+   * Generates the font size utilities based on the
+   * font size list that includes tailwind's default font size values and sensible defaults.
+   *
+   * @returns object - A object with the font size utilities
```

Comment:
```suggestion
   * @returns {object) - A object with the font size utilities
```
Put the type inside `{}`